src/databricks/labs/ucx/installer/policy.py

```diff
@@ -0,0 +1,131 @@
+import json
+import logging
+
+from databricks.labs.blueprint.installation import Installation
+from databricks.labs.blueprint.installer import InstallState
+from databricks.labs.blueprint.tui import Prompts
+from databricks.sdk import WorkspaceClient
+from databricks.sdk.errors import NotFound
+from databricks.sdk.service import compute
+
+logger = logging.getLogger(__name__)
+
+
+class ClusterPolicyInstaller:
+    def __init__(self, installation: Installation, ws: WorkspaceClient, prompts: Prompts):
+        self._ws = ws
+        self._installation = installation
+        self._prompts = prompts
+
+    @staticmethod
+    def _policy_config(value: str):
+        return {"type": "fixed", "value": value}
+
+    def create(self, inventory_database: str) -> tuple[str | None, str | None, dict | None]:
+        instance_profile = None
+        spark_conf_dict = {}
+        policies_with_external_hms = list(self._get_cluster_policies_with_external_hive_metastores())
+        if len(policies_with_external_hms) > 0 and self._prompts.confirm(
+            "We have identified one or more cluster policies set up for an external metastore"
+            "Would you like to set UCX to connect to the external metastore?"
+        ):
+            logger.info("Setting up an external metastore")
+            cluster_policies = {conf.name: conf.definition for conf in policies_with_external_hms}
+            if len(cluster_policies) >= 1:
+                cluster_policy = json.loads(self._prompts.choice_from_dict("Choose a cluster policy", cluster_policies))
+                instance_profile, spark_conf_dict = self._extract_external_hive_metastore_conf(cluster_policy)
+        policy_name = f"Unity Catalog Migration ({inventory_database}) ({self._ws.current_user.me().user_name})"
+        policies = self._ws.cluster_policies.list()
+        for policy in policies:
+            if policy.name == policy_name:
+                logger.info(f"Cluster policy {policy_name} already present, reusing the same.")
+                return policy.policy_id, instance_profile, spark_conf_dict
+        logger.info("Creating UCX cluster policy.")
+        return (
+            self._ws.cluster_policies.create(
+                name=policy_name,
+                definition=self._definition(conf=spark_conf_dict, instance_profile=instance_profile),
```

Comment:
```suggestion
                definition=self._definition(spark_conf_dict, instance_profile),
```
why do you need to use keyed args?